app/renderer/components/sectionTitle.js

```diff
@@ -0,0 +1,131 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const React = require('react')
+const ImmutableComponent = require('../../../js/components/immutableComponent')
+
+const {StyleSheet, css} = require('aphrodite/no-important')
+const globalStyles = require('./styles/global')
+const commonStyles = require('./styles/commonStyles')
+
+class SectionTitleWrapper extends ImmutableComponent {
```

Comment:
Sweet, because I'm still a fan of about:styles, linking to that file from about:styles looks good finally, in my opinion.

Waiting for opinion of other members who will be back online after the holiday.